astropy/io/misc/asdf/tags/transform/tests/test_transform.py

```diff
@@ -283,20 +232,3 @@ def test_fix_inputs_type():
         'compound': astmodels.Pix2Sky_TAN() & {'x': 45}
         }
         helpers.assert_roundtrip_tree(tree, tmpdir)
-
-
-comp_model = custom_and_analytical_inverse()
-
-
-@pytest.mark.parametrize(('model'), [astmodels.Shift(1) & astmodels.Shift(2) | comp_model,
```

Comment:
This test seems important, it's asserting that the inverse is preserved when models are combined.